ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -199,6 +200,15 @@ public void setNumLines(ReactEditText view, int numLines) {
     view.setLines(numLines);
   }
 
+  @ReactProp(name = "maxLength")
+  public void setMaxLength(ReactEditText view, @Nullable Integer maxLength) {
+    if(maxLength != null) {
```

Comment:
Fix spacing:
```
if (maxLength != null) {
```

Also: you need to handle the case when length is null (property gets removed the component) - in that case you'd need to remove filers you set here